File: CS50 HARVARD - Setare Nassaji Matin/ubuntu/pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>
int main(void)
{
    int height;
    //height between 1 and 8
    do
    {
        height = get_int("height: ");
    }
    while (height <= 0 || height > 8 );
    //lets draw 
 for (int row = 0; row < height; row++)
{
    for (int spaces = row + 1; spaces < height; spaces++)
    {
        printf(" ");
    }

    for (int hashes = height + row + 1; hashes > height; hashes--)
    {
        printf("#");
    }
    printf("\n");
}                                                           ;
                                                                              }

File: CS50 HARVARD - Setare Nassaji Matin/ubuntu/pset6/mario.py/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break
for j in range(height):

    # prints spaces (first we want space)
    for spaces in range(height-j-1):
        print(" ", end="")
    # (then hashes)
    for i in range(j+1):
        print("#", end="")


    print("\n", end ="")